docs/hello_nextflow/02_hello_world.md

```diff
@@ -327,85 +311,46 @@ executor >  local (1)
 [46/e4ff05] process > sayHello [100%] 1 of 1 ✔
 ```
 
-Follow the same procedure as before to find the `output.txt` output file. If you want to convince yourself that the parameter is working as intended, feel free to repeat this step with a different output filename.
-
-!!! warning
-
-    If you forget to add the output filename parameter, you get a warning and the output file is called `null`. If you add it but don't give it a value, the output file is called `true`.
+Nextflow will have created a folder called `results/`. In this folder is our `output.txt` file. If you check the contents it should match our existing output. This is how we move results files outside of the working directories.
 
-!!! tip
+### Takeaway
 
-    Command-line arguments take one dash (-) for Nextflow options, two dashes (--) for workflow parameters.
+You know how to use the publishDir directive to move files outside of the Nextflow working directory.
 
-### Takeaway
+!!! note
 
-You know how to use a command line parameter to set the output filename.
+    publishDir has been the primary method of outputting files in Nextflow for a considerable amount of time. However, there is a new syntax that publishes files from the workflow level. Over time we expect this method to become the more prevalent method but we teach publishDir here so you can read older pipelines. As an extension challenge, give the new syntax a try, you can find the documentation here: https://www.nextflow.io/docs/latest/workflow.html#publishing-outputs
 
 ### What's next?
 
-Learn how to set a default value in case we leave out the parameter.
+Learn how to use resume to repeat re-use cached results
 
 ---
 
-## 5. Set a default value for a command line parameter
+## 5. Using the Nextflow resume feature
```

Comment:
Should be "Use" (not "Using") for consistency. These are direct instructions, not topics.